src/components/DropdownMenu/DropdownMenuPopup.tsx

```diff
@@ -160,7 +167,11 @@ export const DropdownMenuPopup = <T,>({
                                 key={index}
                                 className={cnDropdownMenu(
                                     'menu-item',
-                                    {separator: isSeparator(item)},
+                                    {
+                                        separator: isSeparator(item),
+                                        'active-parent': isActiveParent,
```

Comment:
I suppose we should add some information to readme about this classes, otherwise there is no possibility to get it

Reply:
Sure, updated the component's README